frontend/javascripts/components/brain_spinner.tsx

```diff
@@ -106,7 +106,7 @@ export function CoverWithLogin({ onLoggedIn }: { onLoggedIn: () => void }) {
         }}
         align="middle"
       >
-        <Col span={8}>
+        <Col xs={22} sm={20} md={16} lg={12} xl={8}>
           <h3>Try logging in to view the dataset.</h3>
           <LoginForm layout="horizontal" onLoggedIn={onLoggedIn} />
```

Comment:
But the `<CoverWithLogin/>` component is already adjusted right here

To me it looks like this: 
![image](https://github.com/scalableminds/webknossos/assets/39529669/4bad1664-e60f-415b-9354-2e32e7e9a0d6)
![image](https://github.com/scalableminds/webknossos/assets/39529669/4bad1664-e60f-415b-9354-2e32e7e9a0d6)

Reply:
For record: The image shows the `<LoginView/>` component. To get the `<CoverWithLogin />` component rendered, try to open a non public dataset in view mode without being logged in